torch/distributed/distributed_c10d.py

```diff
@@ -1465,6 +1466,128 @@ def reduce_scatter(output,
         work.wait()
 
 
+def all_to_all(output,
+               input,
+               output_split_sizes=[],
+               input_split_sizes=[],
```

Comment:
Lint failures on these two lines are real:

```
  {
    path: 'torch/distributed/distributed_c10d.py',
    start_line: 1471,
    end_line: 1471,
    start_column: 35,
    end_column: 35,
    annotation_level: 'failure',
    message: '[B006] Do not use mutable data structures for argument defaults.  They are created during function definition time. All calls to the function reuse this one instance of that data structure, persisting changes between them.'
  },
  {
    path: 'torch/distributed/distributed_c10d.py',
    start_line: 1472,
    end_line: 1472,
    start_column: 34,
    end_column: 34,
    annotation_level: 'failure',
    message: '[B006] Do not use mutable data structures for argument defaults.  They are created during function definition time. All calls to the function reuse this one instance of that data structure, persisting changes between them.'
  }
```

Reply:
Will fix it to use `None` instead of `[]` as default value.

Reply:
Fixed!